localtypings/pxtcustomeditor.d.ts

```diff
@@ -0,0 +1,186 @@
+declare namespace pxt.extension {
+    interface Message {
```

Comment:
we already have a request/response message system for the editor host. We should reuse it.